Math.md

```diff
@@ -0,0 +1,96 @@
+##Float and Double
+You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.  
+
+Strings may form the backbone of displaying instructions and values to the user, but numbers (and especially mathematical operations on those numbers) are necessary for actually drawing that text to the screen, as well as pretty much every thing the operating system and your applications do, from computing how many days are left in a subscripton to centering a button in window.  
+
+Another kind of number are Float and Double, which represent decimal values such as 0.5 or 3.14159.  The difference between Float and Double is magnitude and precision: Floats are represented in 32 bits and Doubles are represented in 64 bits, so Doubles have the potential for greater magnitude and precision.
+
```

Comment:
Jim,

For updates do I do the complete add/commit/push sequence or is it
different?


   Guy


On Wed, Jun 22, 2016 at 10:16 AM, Jim Campagno <notifications@github.com>
wrote:

> In Math.md
> <https://github.com/learn-co-curriculum/swift-math-readme/pull/3#discussion_r68060575>
> :
>
> > @@ -0,0 +1,96 @@
> > +##Float and Double
> > +You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.
> > +
> > +Strings may form the backbone of displaying instructions and values to the user, but numbers (and especially mathematical operations on those numbers) are necessary for actually drawing that text to the screen, as well as pretty much every thing the operating system and your applications do, from computing how many days are left in a subscripton to centering a button in window.
> > +
> > +Another kind of number are Float and Double, which represent decimal values such as 0.5 or 3.14159.  The difference between Float and Double is magnitude and precision: Floats are represented in 32 bits and Doubles are represented in 64 bits, so Doubles have the potential for greater magnitude and precision.
> > +
>
> This is really good, I would also add a note here that states the
> following:
>
> Taken from Apples Documentation:
> Double has a precision of at least 15 decimal digits, whereas the
> precision of Float can be as little as 6 decimal digits. The appropriate
> floating-point type to use depends on the nature and range of values you
> need to work with in your code. In situations where either type would be
> appropriate, Double is preferred.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-co-curriculum/swift-math-readme/pull/3/files/672c8560f837acb593a383f2c60f8930da795ed0#r68060575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAhDWbaushJhUL517esHgZyYvoxGhlvhks5qOUOjgaJpZM4I7xX4>
> .
>